libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Storage/KeyValueEncryptedFileStore.swift

```diff
@@ -366,28 +413,92 @@ public class KeyValueEncryptedFileStore: NSObject {
         do {
             return try FileManager.default.contentsOfDirectory(atPath: directory)
         } catch {
-            SFSDKCoreLogger.e(KeyValueEncryptedFileStore.self, message: "\(function): Error getting contents of directory: \(error)")
+            SFSDKCoreLogger.e(KeyValueEncryptedFileStore.self, message: "\(callingFunction): Error getting contents of directory: \(error)")
             return []
         }
     }
+    
+    @discardableResult
+    private func writeFile(key: String, value: String, fileType: FileType, callingFunction: String = #function) -> Bool {
```

Comment:
`writeFile`, `removeFile` take a `key`+`fileType` and compute the fileURL but `readFile` takes a `fileURL`.
For symmetry, could all three take a `fileURL` (and leave it to their caller to compute fileURL from key+fileType)?

Reply:
Updated those methods, check out the changes.